specs/wp-manage-domains-spec.js

```diff
@@ -283,4 +291,189 @@ describe( `[${ host }] Managing Domains: (${ screenSize }) @parallel`, function(
 			return await TransferDomainPrecheckPage.Expect( driver );
 		} );
 	} );
+
+	describe.only( 'Register (and cancel) .live domain to an existing site ', function() {
+		const blogName = dataHelper.getNewBlogName();
+		const domainEmailAddress = 'toptest.gfwjyx9f@mailosaur.io';
+		const expectedDomainName = blogName + '.live';
+		const testDomainRegistarDetails = dataHelper.getTestDomainRegistarDetails( domainEmailAddress );
+
+		before( async function() {
+			if ( process.env.SKIP_DOMAIN_TESTS === 'true' ) {
+				await SlackNotifier.warn(
+					'Domains tests are currently disabled as SKIP_DOMAIN_TESTS is set to true',
+					{ suppressDuplicateMessages: true }
+				);
+				return this.skip();
+			}
+		} );
+
+		before( async function() {
+			return await driverManager.ensureNotLoggedIn( driver );
+		} );
+
+		step( 'We can visit set the sandbox cookie for payments', async function() {
```

Comment:
Sandbox cookie is used like on several different places, but I'm not sure is it set properly to purchase `.live` domain. 
As you can see in the test run purchasing `.live` domain is passing. But when user want to cancel and refund it there is no option for that. 

<img width="729" alt="screen shot 2018-09-04 at 16 20 33" src="https://user-images.githubusercontent.com/7116222/45053620-25766f80-b093-11e8-838f-027b6cc5aaaf.png">